test/e2e/kubectl/kubectl.go

```diff
@@ -579,7 +579,10 @@ var _ = SIGDescribe("Kubectl client", func() {
 				if !strings.Contains(ee.String(), "timed out") {
 					framework.Failf("Missing expected 'timed out' error, got: %#v", ee)
 				}
-				framework.ExpectNoError(e2epod.WaitForPodToDisappear(ctx, f.ClientSet, ns, "failure-3", labels.Everything(), 2*time.Second, wait.ForeverTestTimeout))
+				// A single wait.ForeverTestTimeout = 30s was
+				// too short and usually timed out. Found after
+				// df5d84ae811b added error checking.
+				framework.ExpectNoError(e2epod.WaitForPodToDisappear(ctx, f.ClientSet, ns, "failure-3", labels.Everything(), 2*time.Second, 2*wait.ForeverTestTimeout))
```

Comment:
```suggestion
				framework.ExpectNoError(e2epod.WaitForPodToDisappear(ctx, f.ClientSet, ns, "failure-3", labels.Everything(), 2*time.Second, 2*v1.DefaultTerminationGracePeriodSeconds*time.Second))
```

this depends on the gracedperiod IIUIC

Reply:
I've verified that this works

Reply:
Changed. I also removed my comment because it no longer applied.